2996_reflection/reflection.md

```diff
@@ -3696,66 +3840,60 @@ Extend the grammar for `$using-enum-declarator$` as follows:
 :::
 
 ### [namespace.alias]{.sref} Namespace alias {-}
 
-Add a production to the grammar for `$qualified-namespace-specifier$` as follows:
+Add a production to the grammar for `$namespace-alias-definition$` as follows:
 
 ::: std
 ```diff
-  $namespace-alias$:
-      $identifier$
-
   $namespace-alias-definition$:
       namespace $identifier$ = $qualified-namespace-specifier$
-
-  $qualified-namespace-specifier$:
-      $nested-name-specifier$@~_opt_~@ $namespace-name$
-+     $splice-specifier$
++     namespace $identifier$ = $splice-specifier$
 ```
 :::
 
 Add the following prior to paragraph 1, and renumber accordingly:
 
 ::: std
 :::addu
-[0]{.pnum} If a `$qualified-namespace-specifier$` is a `$splice-specifier$`, the `$splice-specifier$` shall designate a namespace or namespace alias; the `$qualified-namespace-specifier$` designates the same namespace or namespace alias designated by the `$splice-specifier$`. Otherwise, the `$qualified-namespace-specifier$` designates the namespace found by lookup ([basic.lookup.unqual]{.sref}, [basic.lookup.qual]{.sref}).
+[0]{.pnum} A `$splice-specifier$` appearing in a `$namespace-alias-definition$` shall designate a namespace or `$namespace-alias$`.
 :::
 :::
 
-Prefer the verb "designate" for `$qualified-namespace-specifiers$` in the paragraph that immediately follows:
+Prefer the verb "designate" in the paragraph that immediately follows:
 
 ::: std
-[2]{.pnum} The `$identifier$` in a `$namespace-alias-definition$` becomes a `$namespace-alias$` and denotes the namespace [denoted]{.rm} [designated]{.addu} by the `$qualified-namespace-specifier$`.
+[2]{.pnum} The `$identifier$` in a `$namespace-alias-definition$` becomes a `$namespace-alias$` and denotes the namespace [denoted]{.rm} [designated]{.addu} by the `$qualified-namespace-specifier$` [or `$splice-specifier$`]{.addu}.
 :::
 
 ### [namespace.udir]{.sref} Using namespace directive {-}
 
-Use `$qualified-namespace-specifier$` in the grammar for `$using-directive$`:
+Add `$splice-specifier$` to the grammar for `$using-directive$`:
 
 ::: std
 ```diff
   $using-directive$:
--    $attribute-specifier-seq$@~_opt_~@ using namespace $nested-name-specifier$@~_opt_~@ $namespace-name$
-+    $attribute-specifier-seq$@~_opt_~@ using namespace $qualified-namespace-specifier$
+      $attribute-specifier-seq$@~_opt_~@ using namespace $nested-name-specifier$@~_opt_~@ $namespace-name$
++     $attribute-specifier-seq$@~_opt_~@ using namespace $splice-specifier$
 ```
 :::
 
 Add the following prior to the first paragraph of [namespace.udir]{.sref}, and renumber accordingly:
 
 ::: std
 ::: addu
-[0]{.pnum} The `$qualified-namespace-specifier$` shall neither contain a dependent `$nested-name-specifier$` nor a dependent `$splice-specifier$`.
+[0]{.pnum} The `$splice-specifier$`, if any, designates a namespace or `$namespace-alias$`. Neither the `$nested-name-specifier$` nor the `$splice-specifier$` shall be dependent.
```

Comment:
I think flip it - The nested-name-specifier and the splice-specifier shall not be dependent.

Reply:
Done.